test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -197,5 +197,41 @@ public void NgramWorkout()
             CheckEquality("Text", "ngrams.tsv");
             Done();
         }
+
+        [Fact(Skip = "Despite seed is set, LDA is giving different vector at every run. Need to fix it.")]
+        public void LdaWorkout()
+        {
+            string sentimentDataPath = GetDataPath("wikipedia-detox-250-line-data.tsv");
+            var data = TextLoader.CreateReader(Env, ctx => (
```

Comment:
>Env [](start = 47, length = 3)

@zeahmed   LDA will produce different results for different amount of threads, so for this test I would suggest create your own Environment with num_threads = 1.

Reply:
There is a parameter call `ResetRandomGenerator` that's controlling the randomness. I fixed it.

---
In reply to: [219916345](https://github.com/dotnet/machinelearning/pull/972#discussion_r219916345) [](ancestors = 219916345)

Reply:
It's still failing on build machine. As I said, you can play with num_threads for enviroment, and check did you get same results or not. If not, just set num_threads for this test to 1.

---
In reply to: [219921802](https://github.com/dotnet/machinelearning/pull/972#discussion_r219921802) [](ancestors = 219921802,219916345)

Reply:
As suggested by @tfinley@gmail.com, I have disabled the full output file checking in the test. Also, opened issue #1004 to track it.

---
In reply to: [219965528](https://github.com/dotnet/machinelearning/pull/972#discussion_r219965528) [](ancestors = 219965528,219921802,219916345)